src/main/scala/nl/knaw/dans/easy/fsrdb/package.scala

```diff
@@ -24,7 +24,8 @@ import scala.util.{Success, Failure, Try}
 package object fsrdb {
   case class Settings(fedoraCredentials: FedoraCredentials,
                       postgresURL: String,
-                      datasetPid: String)
+                      input: Option[File] = None,
+                      datasetPids: Option[List[String]] = None)
```

Comment:
Easy-ingest-flow will need a backward compatible constructor or a factory method (apply method in the companion object)
See http://stackoverflow.com/questions/2400794/overload-constructor-for-scalas-case-classes

other easy projects use the apply methods, one with a a backward compatible signature for ingest-flow or ingest-dispatcher, one with some ScallopConf argument and perhaps the properties to be called by the main method.